LiskHQ/lisk-sdk
# Update max height previously forged - Closes #3696

### What was the problem?
- Max height previously forged was not persisted when creating a block
- Wrong maxPreviouslyForgedHeight was used while creating a block

### How did I solve it?
- Add a function to save previously forged height per delegate
- Use the saved value for BFT header

### How to manually test it?
- Start the node and observe chain_meta table that maxHeightPreviouslyForged is saved
- Check that newly created block has the correct previously forged height per delegate

### Review checklist

- [ ] The PR resolves #3696
